experimental/alp/alp/mlirc.py

```diff
@@ -0,0 +1,57 @@
+import sys
```

Comment:
Same comment re using the harness and metaprogramming IR from python in the future: this is really hacky (fine for a first commit).